jenkins_jobs/build_host_os/script.sh

```diff
@@ -32,5 +35,9 @@ python host_os.py \
 # creating the yum repository locally
 createrepo ./repository
 
+# Create META file with timestamp information
+touch ./META
+echo "${TIMESTAMP}" > ./META
```

Comment:
I'd prefer to name it TIMESTAMP or BUILD_TIMESTAMP. META may make one think there's more information in this file.